### PR TITLE
update README

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

**Review by johan-da — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -94,6 +97,6 @@ When there is a change to the root website, the entire site needs to be reindexe
 
 ## Questions
 
-If you have any questions or comments about these instructions, please reach out to the `#product-docs` channel and/or @katharine/@gary on Slack.
+If you have any questions or comments about these instructions, please reach out to the `#product-docs` channel, mentionning @gary, on Slack.
```
Comment (loiskilberg-da) [inline README.md:100]:
```suggestion
If you have any questions or comments about these instructions, please reach out to the `#product-docs` channel, mentioning @gary, on Slack.
```

**Review by loiskilberg-da — APPROVED** [review]
Looks good! just one spelling suggestion

File: README.md (hunk 1 above)
Comment (garyverhaegen-da) [reply README.md]: Thanks, corrected.

**Pull request merged**